### DISCUSSION TITLE
Serveurs physiques, virtuels ou containers.

### DISCUSSION THREAD

**Discussion by Gouagou** [dbarzin/mercator#958] [Ideas]
Bonjour,

Dans la liste des serveurs logiques, comment pouvons-nous différencier les différents types de serveurs (physique, virtuel, container, ...) ?
Est-ce possible d’ajouter un champ spécifique ?

**Comment by dbarzin** [comment]
Un champ type va être ajouté dans la table des serveurs logiques. Le type sera également présent dans la liste.

**Comment by dbarzin** [comment]
:tada: C'est disponible : https://github.com/dbarzin/mercator/pull/959

**Reply by Gouagou** [reply]
Merci :smiley:
C'est Noël avant l'heure !
